zk-token-sdk/src/lib.rs

```diff
@@ -1,3 +1,9 @@
+// Deprecate the crate
+#![deprecated(
+    note = "use the `zk-sdk` instead: https://github.com/solana-program/zk-elgamal-proof/tree/main/zk-sdk"
```

Comment:
nit: sorry, can this be totally explicit?
```suggestion
    note = "use the `solana-zk-sdk` instead: https://github.com/solana-program/zk-elgamal-proof/tree/main/zk-sdk"
```